src/utils/format-string.ts

```diff
@@ -0,0 +1,8 @@
+export function formatString(inputString: string | undefined, maxLength: number): string {
```

Comment:
where do we use this?
I think you can use this [tailwindcss feature](https://tailwindcss.com/docs/text-overflow) 🤔

Reply:
Okey thanks :3 i didn't know there is a class for this hehe